ukrainian/rope/game-over/game-over.md

```diff
@@ -0,0 +1,3 @@
+Гру скінчено!
+
+[Розпочати спочатку?](marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.